homeassistant/components/zoneminder.py

```diff
@@ -40,6 +41,29 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+class _Feature:
+    def __init__(self, version_str):
+        self._version = packaging.version.parse(version_str)
+
+    def is_supported(self):
+        """Return the remote instance support for this feature.
+
+        Returns False if this feature isn't supported on the currently
+        connected zoneminder instance. Otherwise returns True.
+        """
+        remote_version = ZM.get('version', None)
```

Comment:
I wanted to avoid adding a reference to `zoneminder.ZM` in the `camera.zoneminder` module and thought that adding a class would make it easier to support additional features. How about I add a `zoneminder.alarm_status_supported()` function for `camera.zoneminder` to call?

I'm happy to do the refactoring to get rid of the ZM object. Is there a different module that I can use as a good reference? Would move `zoneminder.ZM` to `hass.data['zoneminder']` like `nest` be the thing to do?